roles/pulp_common/defaults/main.yml

```diff
@@ -15,7 +15,7 @@ pulp_upgrade: false
 #  they are confident that this version of pulp_installer is compatible with
 #  pulp_version.
 #  Ignored if pulp_source_dir is set
-pulp_version: "3.8.1"
+__pulp_version: '{{ pulp_version | default("3.8.1") }}'
```

Comment:
We can remove the above comment now (all 4 lines.)

Reply:
Done.